Package.swift

```diff
@@ -17,7 +17,8 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-llbuild.git", .branch("master")),
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.8.0"),
         .package(url: "https://github.com/apple/swift-tools-support-core.git", from: "0.0.1"),
-        .package(url: "https://github.com/apple/swift-protobuf.git", .branch("master")),
+        .package(url: "https://github.com/apple/swift-protobuf.git", from: "1.8.0"),
```

Comment:
@sergiocampama FYI I moved protobuf from master to 1.8.0..<2.0.0 as that's what grpc-swift's master depends on.